Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.2 as build
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3 as build
```

Comment:
RH doesn't do point releases such as 9.2 to 9.2.x instead they go to 9.3 it is expected that ubi9 is the main image and they are compatible with each other since the major version doesn't change.

We can just use `:latest` always whenever we release @pjuarezd https://github.com/minio/minio/pull/18497